sonic-slave-buster/Dockerfile.j2

```diff
@@ -530,6 +630,18 @@ RUN pip3 install git+https://github.com/aristanetworks/swi-tools.git@bead66bf261
 RUN curl -sL https://deb.nodesource.com/setup_14.x | bash -
 RUN apt-get install -y nodejs
 
+{%- if CROSS_BUILD_ENVIRON == "y" %}
```

Comment:
You should install these packages, RUN apt-get install libgtest-dev:$arch libgmock-dev:$arch -y # For sonic-swss-common tests

if not the sonic-swss-common unit tests are failing